### PR TITLE
pls

### PR DESCRIPTION
**Pull request by Ryang20718**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,10 @@
 
 *Owned by tfukaza, Feb 4 2022*
 
+## yellow
+![yellow](https://singlecolorimage.com/get/A020F0/400x100)
```
Comment (sean1006) [inline README.md:25]: I think you could check this out: [Comprehensive Markdown Style Guide](https://seesawseeneyewear.com/)

**Review by sean1006 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Ryang20718) [reply README.md:25]: no u

**Comment by Kimeiga** [comment]
Wait don't you need to sign your attribution lol

**Comment by Kimeiga** [comment]
wait nvm it wasn't showing on mobile browser

**Pull request merged**